python/pyenv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
```

Comment:
you should be using the [`github` PG](https://guide.macports.org/index.html#reference.portgroup.github)

Reply:
_This_ makes everything make more sense, thank you.